### DISCUSSION TITLE
Just thanks

### DISCUSSION THREAD

**Discussion by Vidyut** [peter88213/yw2html#3] [General]
Nothing to say, just thanks. I just lurk on the group, but the one time I had a problem ages ago, you posted this and solved it. Wanted you to know I'm still using it. Thank you.

**Comment by peter88213** [comment]
You're welcome. Glad to know that. 
Your request for a template-based HTML export was a great inspiration. 
So I wrote a general purpose template-engine to export any format from the yWriter data model, such as Markdown and all sorts of ODT and ODS, even DOCX.

By the way, I just opened "Discussions" for general fedback. 

Cheers, 
Peter

**Reply by Vidyut** [reply]
That's interesting. Is this a new project? Can you link to it or give its name? Or is this new functionality I hadn't explored?

**Reply by peter88213** [reply]
The programs are listed on https://peter88213.github.io/ under "The yWriter toolbox". 
*yw2xtg*, for instance is very similar to *yw2html*, but it works with XTG-coded custom templates instead of HTML.
Most of the other tools, like *yw2OO* or *yw2md*, have pre-made templates built in that are tailored for several document types. 
With *yw_reporter* it is easy to configure which of the built-in templates are to be used.

Enjoy.
